### PR TITLE
Convert noteblocks for web/html folder

### PR DESCRIPTION
**Pull request by queengooborg**
This PR converts the noteblocks for the 'web/html' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

### CODE REVIEW

File: files/en-us/web/html/element/a/index.md [hunk 1]
```diff
@@ -51,7 +52,9 @@ This element's attributes include the [global attributes](/en-US/docs/Web/HTML/G
 
     - `filename`: defining a value suggests it as the filename. `/` and `\` characters are converted to underscores (`_`). Filesystems may forbid other characters in filenames, so browsers will adjust the suggested name if necessary.
 
-    > **Note:**
+    > [!NOTE]
+    >
+    > >
```
Comment (Josh-Cena) [inline files/en-us/web/html/element/a/index.md:57]:
```suggestion
    > [!NOTE]
```

File: files/en-us/web/html/element/audio/index.md [hunk 2]
```diff
@@ -67,7 +68,9 @@ This element's attributes include the [global attributes](/en-US/docs/Web/HTML/G
 
     The default value is different for each browser. The spec advises it to be set to `metadata`.
 
-    > **Note:**
+    > [!NOTE]
+    >
+    > >
```
Comment (Josh-Cena) [inline files/en-us/web/html/element/audio/index.md:73]:
```suggestion
    > [!NOTE]
```

@queengooborg FYI

File: files/en-us/web/html/element/iframe/index.md [hunk 3]
```diff
@@ -125,24 +129,29 @@ This element includes the [global attributes](/en-US/docs/Web/HTML/Global_attrib
     - `allow-top-navigation-to-custom-protocols`
       - : Allows navigations to non-`http` protocols built into browser or [registered by a website](/en-US/docs/Web/API/Navigator/registerProtocolHandler/Web-based_protocol_handlers). This feature is also activated by `allow-popups` or `allow-top-navigation` keyword.
 
-    > **Note:**
+    > [!NOTE]
+    >
+    > >
```
Comment (Josh-Cena) [inline files/en-us/web/html/element/iframe/index.md:134]:
```suggestion
    > [!NOTE]
```

File: files/en-us/web/html/element/meta/index.md [hunk 4]
```diff
@@ -54,9 +54,9 @@ This element includes the [global attributes](/en-US/docs/Web/HTML/Global_attrib
 
       The timer starts when the page is _completely loaded_, which is after the {{domxref("Window/load_event", "load")}} and {{domxref("Window/pageshow_event", "pageshow")}} events have both fired.
 
-      > **Warning:**
+      > [!WARNING]
       >
-      > Pages set with a `refresh` value run the risk of having the time interval being too short. People navigating with the aid of assistive technology such as a screen reader may be unable to read through and understand the page's content before being automatically redirected. The abrupt, unannounced updating of the page content may also be disorienting for people experiencing low vision conditions.
+      > > Pages set with a `refresh` value run the risk of having the time interval being too short. People navigating with the aid of assistive technology such as a screen reader may be unable to read through and understand the page's content before being automatically redirected. The abrupt, unannounced updating of the page content may also be disorienting for people experiencing low vision conditions.
```
Comment (Josh-Cena) [inline files/en-us/web/html/element/meta/index.md:59]:
```suggestion
      > Pages set with a `refresh` value run the risk of having the time interval being too short. People navigating with the aid of assistive technology such as a screen reader may be unable to read through and understand the page's content before being automatically redirected. The abrupt, unannounced updating of the page content may also be disorienting for people experiencing low vision conditions.
```

File: files/en-us/web/html/element/video/index.md [hunk 5]
```diff
@@ -77,7 +78,9 @@ Like all other HTML elements, this element supports the [global attributes](/en-
 
     The default value is different for each browser. The spec advises it to be set to `metadata`.
 
-    > **Note:**
+    > [!NOTE]
+    >
+    > >
```
Comment (Josh-Cena) [inline files/en-us/web/html/element/video/index.md:83]:
```suggestion
    > [!NOTE]
```

File: files/en-us/web/html/element/audio/index.md (hunk 2 above)
Comment (queengooborg) [reply files/en-us/web/html/element/audio/index.md:73]: Okay, I _swear_ I am catching these before creating the PRs...  I wonder if something's up with my local Husky hooks, hmm... 🤔

**Review by Josh-Cena — APPROVED** [review]

**Pull request merged**